src/locales/default.json

```diff
@@ -317,6 +317,7 @@
   "mustBeValidEmail": "must be a valid email",
   "mustBeValidNum": "must be a valid number",
   "mustBeGreaterThan": "must be greater than {0}",
+  "mustBeLessThanChars": "must be less than {0} characters",
```

Comment:
You added a translation here but aren't using it anywhere

Reply:
mustBeLessThanChars was already being use on line 15 of src\beethovenx\utils\validations.ts. I added it to the default.json so it will get displayed correctly.  Screenshot in from production:

![image](https://user-images.githubusercontent.com/99622829/163894628-90104ad5-38b0-41b2-bbfe-bab15a0596f7.png)

Reply:
gotcha, looks good